### PR TITLE
Implement standorte

### LINKED ISSUE #23: [Site] Locations page
See https://www.eder-gmbh.de/unternehmen/standorte/

- [ ] Implement location list
- [ ] Implement Google Maps
- [ ] Implement static (configured via block config) filters
- [ ] Implement post code search for map

### PR DESCRIPTION
**Pull request by alexcarol**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #23

Test URLs:
- Original: https://www.eder-gmbh.de/unternehmen/standorte/
- Before: https://main--eder-group--hlxsites.hlx.page/unternehmen/standorte
- After: https://issue-23--eder-group--hlxsites.hlx.page/unternehmen/standorte

Missing:
- [will create new issue for this] Animate list elements when being (un)filtered
- [discussed optional, needs clarification from customer] Use [spiderfeier](https://github.com/jawj/OverlappingMarkerSpiderfier/) to avoid google maps marker stacking
- [needs clarification from customer] Create postcode filter

### CODE REVIEW

File: blocks/standorte/standorte.css [hunk 1]
```diff
@@ -1,11 +1,8 @@
-.standorte {
-    background-color: white;
-}
-
 .standorte > div {
+    background-color: white;
```
Comment (mhaack) [inline blocks/standorte/standorte.css:2]: Use color variables here.

File: blocks/standorte/standorte.css [hunk 2]
```diff
@@ -45,6 +42,49 @@
     margin: 0;
 }
 
+.standorte-container .default-content-wrapper h2 {
+    background-color: white;
```
Comment (mhaack) [inline blocks/standorte/standorte.css:46]: Use color variables here.

File: blocks/standorte/standorte.css [hunk 3]
```diff
@@ -45,6 +42,49 @@
...
+    margin: 0;
+    padding: 30px 30px 20px;
+    text-transform: uppercase;
+    font-weight: bold;
+    font-size: 24px;
```
Comment (mhaack) [inline blocks/standorte/standorte.css:51]: Use variables for font-size.

File: blocks/standorte/standorte.css [hunk 4]
```diff
@@ -45,6 +42,49 @@
...
+}
+
+.standorte-map ul {
+    list-style: none;
+    padding: 20px;
+}
+
+.standorte-map li {
+    display: inline-block;
+    color: #e5e2db;
```
Comment (mhaack) [inline blocks/standorte/standorte.css:66]: Use color variables here.

File: blocks/standorte/standorte.js [hunk 5]
```diff
@@ -1,7 +1,149 @@
...
+
+      bereicheFilterList.append(li);
+    });
+
+    filterDiv.append(bereicheFilterList);
+
+    mapDiv.append(mapElement, filterDiv);
+    block.parentElement.prepend(mapDiv);
+
+    const script = document.createElement('script');
```
Comment (mhaack) [inline blocks/standorte/standorte.js:104]: We have `loadScript` helper for this

File: blocks/standorte/standorte.css (hunk 4 above)
Comment (alexcarol) [reply blocks/standorte/standorte.css:66]: This is the only case where this is used on the site, should I create the variable in styles.css or in this css file?

File: blocks/standorte/standorte.css (hunk 3 above)
Comment (alexcarol) [reply blocks/standorte/standorte.css:51]: Regarding this, should I be creating new variables for this font size specifically for this page? I couldn't find any variables for 24 or 21px (also in this file). I have the feeling that they're not a one-off, would it make sense to create a task to unify variables (size, color, margins and so on), as it requires a high-level view of the page?

File: blocks/standorte/standorte.js (hunk 5 above)
Comment (alexcarol) [reply blocks/standorte/standorte.js]: done

File: blocks/standorte/standorte.css (hunk 3 above)
Comment (mhaack) [reply blocks/standorte/standorte.css:51]: If it is specific to this block only the lets leave it like its is. Ignore my comment.

File: blocks/standorte/standorte.css (hunk 4 above)
Comment (mhaack) [reply blocks/standorte/standorte.css:66]: Would `--background-text` looks pretty close? Oterwhise create a new.

**Review by mhaack — APPROVED** [review]

File: blocks/standorte/standorte.css (hunk 4 above)
Comment (alexcarol) [reply blocks/standorte/standorte.css:66]: Looks close enough, changed!

**Pull request merged**